scripts/install_docker.sh

```diff
@@ -3,3 +3,5 @@ set -ex
 
 curl -sSL https://get.docker.com/ubuntu/ | sh
 sudo gpasswd -a vagrant docker
+docker build -t android-base /vagrant/docker/android-base
+docker build -t android-studio /vagrant/docker/android-studio
```

Comment:
Seems like we should remove these two build lines now that we added the ENV-gated provisioning step